Source/ACE.Server/Managers/PropertyManager.cs

```diff
@@ -482,6 +482,7 @@ public static void LoadDefaultProperties()
 
         public static readonly ReadOnlyDictionary<string, string> DefaultStringProperties =
             DictOf(
+                ("content_folder", ".\\Content"),   // for content creators to live edit weenies. defaults to Content in the netcoreapp2.1 folder
```

Comment:
content_folder seems like a system property, so I think it should go in Config.js, not in the database.

Reply:
config.js is for things that are required to start up the server

content_folder is not required to start up the server. it is also something that won't be used for normal server operation, only for content editing

PropertyManager also gives the user the ability to change content_folder on the fly